src/components/page-search/page-search.test.js

```diff
@@ -65,7 +65,7 @@ describe('heatmap', () => {
   it('loads posts into heatmap via subreddit in URL', async () => {
     setup('/search/reactjs');
 
-    await waitForElementToBeRemoved(() => screen.getByLabelText(/loading/i));
+    await waitForElementToBeRemoved(() => screen.getByLabelText(/loading/i), { timeout: 10000 });
```

Comment:
It is interesting that this takes so long. Are you sure that the mock handlers are called? This looks suspiciously like it's sending real requests